bash_completionsV2.go

```diff
@@ -177,40 +177,28 @@ __%[1]s_handle_standard_completion_case() {
     local tab=$'\t' comp
 
     local longest=0
+    local compline
     # Look for the longest completion so that we can format things nicely
-    while IFS='' read -r comp; do
+    while IFS='' read -r compline; do
         # Strip any description before checking the length
-        comp=${comp%%%%$tab*}
+        comp=${compline%%$tab*}
```

Comment:
Because `%` is a special character in Go's `Sprintf()` function, we need to escape it by making it double, so a single `%` must become `%%` but in our case we need a double `%%` in the final bash script so as to remove the description that follows the first tab character (in case the description contains a tab itself).  So we need to put 4 `%%%%` to get the final double `%%`.